src/main/java/org/alliancegenome/curation_api/config/RestApplication.java

```diff
@@ -3,11 +3,13 @@
 import javax.ws.rs.ApplicationPath;
 import javax.ws.rs.core.Application;
 
+import org.alliancegenome.curation_api.auth.Secured;
 import org.eclipse.microprofile.openapi.annotations.*;
 import org.eclipse.microprofile.openapi.annotations.enums.SecuritySchemeType;
 import org.eclipse.microprofile.openapi.annotations.info.Info;
 import org.eclipse.microprofile.openapi.annotations.security.*;
 
+@Secured
```

Comment:
@mluypaert putting @Secured here "turns it on" for the whole API vs endpoint by endpoint.